### PR TITLE
[Refactor] Grid view for Filters

### PR DESCRIPTION
**Pull request by subhradwip01**
Signed-off-by: subhradwip01 <subhradwip01@gmail.com>

**Description**
Grid view added for meshery filters page
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by YashKamboj — CHANGES_REQUESTED** [review]
Add the pagination  @subhradwip01

File: ui/components/MesheryFilterGrid.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  const [gridProps, setGridProps] = useState(INITIAL_GRID_SIZE);
+  // const [yaml, setYaml] = useState(filetrs.filter_file);
+
+  return (
+    <Grid item {...gridProps}>
+      <MesheryCard
+        name={filter.name}
+        updated_at={filter.updated_at}
+        created_at={filter.created_at}
+        //  file={filter.filetr_file}
```
Comment (YashKamboj) [inline ui/components/MesheryFilterGrid.js:17]: You don't need to comment the file , handledeploy and handleundeploy @subhradwip01

**Review by YashKamboj — CHANGES_REQUESTED** [review]

File: ui/components/MesheryFilterGrid.js (hunk 1 above)
Comment (subhradwip01) [reply ui/components/MesheryFilterGrid.js:17]: Ya ... actually i was going to uncommet thr file but forgot 😅

**Review by YashKamboj — APPROVED** [review]
LGTM!!

File: ui/components/Filters/MesheryFilterGrid.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  filters = [],
+  handleDeploy,
+  handleUnDeploy,
+  handleSubmit,
+  setSelectedFilter,
+  selectedFilter,
+  pages = 1,
+  setPage,
+  selectedPage,
+}) {
```
Comment (Abhishek-kumar09) [inline ui/components/Filters/MesheryFilterGrid.js:62]: They both should match, otherwise the vscode will give the wrong suggestion 😄

**Comment by Abhishek-kumar09** [comment]
Also, post a screenshot of your change

**Pull request merged**